projects/igniteui-angular/src/lib/directives/radio/radio-group.directive.ts

```diff
@@ -340,10 +347,10 @@ export class IgxRadioGroupDirective implements AfterContentInit, ControlValueAcc
      * @hidden
      * @internal
      */
-     public setDisabledState(isDisabled: boolean) {
+    public setDisabledState(isDisabled: boolean) {
         if (this.radioButtons) {
             this.radioButtons.forEach((button) => {
-                button.disabled = isDisabled;
+                button.setDisabledState(isDisabled);
```

Comment:
There's something off with the changes in the PR, can you please re-check if some conflict resolution didn't cause this?